app/templates/create.html

```diff
@@ -0,0 +1,39 @@
+<!-- extend base layout -->
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Create Thread</h1>
+  <form action="" method="post" name="edit">
+      {{ form.hidden_tag() }}
+      <table>
+          <tr>
+              <td>Title:</td>
+              <td>
+                {{ form.title(size=24, maxlength=100, required="True") }}
+                {% for error in form.title.errors %}
+                <span style="color: red;">[{{ error }}]</span><br>
+                {% endfor %}
+              </td>
+              
+          </tr>
+          <tr>
+              <td>Topic:</td>
+              <td>{{ form.topic }}</td>
+              {% for error in form.topic.errors %}
+              <span style="color: red;">[{{ error }}]</span><br>
+              {% endfor %}
+          </tr>
+          <tr>
+              <td>Body:</td>
```

Comment:
Line 27: 
Suggestion -- change 'Body' to 'Description' or any other term that would make the user understand that this field describes what the title of the thread is all about. :smile:

e.g.,
Title: Roses
Topic: Music
Description: This album is by The Chainsmokers.

:smile: